tests/rgw/sanity_rgw_multisite.py

```diff
@@ -182,6 +186,14 @@ def run(**kw):
                 )
             verify_sync_status(copy_user_to_site.get_ceph_object("rgw").node)
 
+        pri_node = clusters.get("ceph-pri").get_ceph_object("rgw").node
+        sec_node = clusters.get("ceph-sec").get_ceph_object("rgw").node
```

Comment:
In this file we already have variable for rgw node
    primary_rgw_node = primary_cluster.get_ceph_object("rgw").node
    secondary_rgw_node = secondary_cluster.get_ceph_object("rgw").node
Do we need to define it again with new variable??

Reply:
Would test with that.

Reply:
@anrao19 , It works with the already defined variables of "primary_rgw_node' and 'secondary_rgw_node'. I tested with it.